### PR TITLE
VSF2-168 - Fixed mobile view appearance

### PR DESCRIPTION
**Pull request by DawidFrankiewicz**

### CODE REVIEW

File: packages/theme/components/AppFooter.vue [hunk 1]
```diff
@@ -94,21 +94,22 @@ export default {
 <style lang="scss">
 
 .footer {
-  margin-bottom: 3.75rem;
-  @include for-desktop {
-    margin-bottom: 0;
+  @include for-mobile{
```
Comment (dead-playmod) [inline packages/theme/components/AppFooter.vue:97]: missing space after `for-mobile`

File: packages/theme/components/AppFooter.vue [hunk 2]
```diff
@@ -94,21 +94,22 @@ export default {
...
+    margin-bottom: 3.75rem;
```
Comment (dead-playmod) [inline packages/theme/components/AppFooter.vue:98]: try using predefined spacer css variables e.g. `var(--spacer-lg)`

File: packages/theme/components/AppFooter.vue [hunk 3]
```diff
@@ -94,21 +94,22 @@ export default {
...
+    padding-bottom: 1rem;
```
Comment (dead-playmod) [inline packages/theme/components/AppFooter.vue:99]: try using predefined spacer css variables e.g. `var(--spacer-lg)`

File: packages/theme/components/FiltersSidebar.vue [hunk 4]
```diff
@@ -312,6 +312,11 @@ export default {
 ::v-deep .sf-sidebar__aside {
   --sidebar-z-index: 3;
 }
+::v-deep .sf-sidebar__bottom {
+  @include for-mobile{
+    margin-bottom: 80px;
```
Comment (dead-playmod) [inline packages/theme/components/FiltersSidebar.vue:317]: try using predefined spacer css variables e.g. `var(--spacer-lg)`

**Review by dead-playmod — CHANGES_REQUESTED** [review]

File: packages/theme/components/AppFooter.vue (hunk 2 above)
Comment (DawidFrankiewicz) [reply packages/theme/components/AppFooter.vue:98]: There is now var that would work here

**Review by dead-playmod — APPROVED** [review]

**Pull request merged**